### PR TITLE
Dhcp Lease List Added

### PR DESCRIPTION
**Pull request by ftoure67**
Signed-off-by: ftoure67 <famory.toure@gmail.com>

Displays the DHCP leases for the interfaces acting as DHCP servers.
Relasted issue: #3755

Screenshots: https://photos.google.com/photo/AF1QipOi_IS7iXku1GKlN8ufQtZB7vhG8k0_HnB0NAEE

### CODE REVIEW

File: kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/DhcpLeaseTool.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
+package org.eclipse.kura.linux.net.util;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/DhcpLeaseTool.java:13]: Please add the copyright header

File: kura/test/org.eclipse.kura.linux.net.test/src/test/java/org/eclipse/kura/linux/net/util/DhcpLeaseToolTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
+package org.eclipse.kura.linux.net.util;
```
Comment (pierantoniomerlino) [inline kura/test/org.eclipse.kura.linux.net.test/src/test/java/org/eclipse/kura/linux/net/util/DhcpLeaseToolTest.java:13]: Please follow the [guide](https://github.com/eclipse/kura/blob/develop/CONTRIBUTING.md) for the test development. Recently we switched to the Gerkin style for new tests. It follows the given-when-then pattern for testing features.

**Comment by pierantoniomerlino** [comment]
@ftoure67 I successfully tested the PR on a RPi. The dhcp leases are displayed on the web console even if the client is disconnected, but I think that this is the standard behavior of the dhcp service on the OS. Indeed the dhcp-leases command returns the listed values.

Please, add the copyright header in the new files and update it in the old ones if needed.
Finally, could you please rebase your branch with develop?

Thanks for you contribution!

**Comment by MMaiero** [comment]
Hi @ftoure67 
do you believe you can address the issues reported by @pierantoniomerlino ?
Once done, I believe we can easily merge the changes

Best regards,
Matteo

File: kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/DhcpLeaseTool.java (hunk 1 above)
Comment (ftoure67) [reply kura/org.eclipse.kura.linux.net/src/main/java/org/eclipse/kura/linux/net/util/DhcpLeaseTool.java:13]: OK.

File: kura/test/org.eclipse.kura.linux.net.test/src/test/java/org/eclipse/kura/linux/net/util/DhcpLeaseToolTest.java (hunk 2 above)
Comment (ftoure67) [reply kura/test/org.eclipse.kura.linux.net.test/src/test/java/org/eclipse/kura/linux/net/util/DhcpLeaseToolTest.java:13]: OK.

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtDhcpLease.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package org.eclipse.kura.web.shared.model;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtDhcpLease.java:13]: Please add the copyright header also here

**Comment by pierantoniomerlino** [comment]
@ftoure67 It seems that there are conflicts with the develop branch. You should rebase your branch with develop and fix the conflicts.

Thanks!

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtDhcpLease.java (hunk 3 above)
Comment (ftoure67) [reply kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtDhcpLease.java:13]: OK.

**Review by pierantoniomerlino — APPROVED** [review]

**Pull request merged**